src/pages/home/report/ReportActionCompose.js

```diff
@@ -471,7 +471,7 @@ class ReportActionCompose extends React.Component {
         const trimmedComment = this.comment.trim();
 
         // Don't submit empty comments or comments that exceed the character limit
-        if (this.state.isCommentEmpty || trimmedComment.length > CONST.MAX_COMMENT_LENGTH) {
+        if (this.state.isCommentEmpty || ReportUtils.commentLength(trimmedComment) > CONST.MAX_COMMENT_LENGTH) {
```

Comment:
@redstar504 We also need to do this in `ReportActionItemMessageEdit` because I still can send the message with press _Enter_.

Reply:
Good call, thanks.